docs/frameworks/sveltekit.md

```diff
@@ -18,49 +18,28 @@ You should remove all references to [SvelteKit service worker module](https://ki
 
 ## SvelteKit Vite Plugin
 
-`Sveltkit` maintainers have been working hard to align `Vite` and `SvelteKit`, right now almost **99%** aligned:
-- `SvelteKit` exposes `publicDir` to allow `Vite` plugins using it: `publicDir` is configured with `config.kit.files.assets` (defaults to `static` folder).
-- `SvelteKit` exposes `outDir` to allow `Vite` plugins using it: `outDir` configured with `${svelteKitOutDir}/output/client` (defaults to `.svelte-kit/output/client` folder).
-
-`vite-plugin-pwa` exposes a new `Vite` plugin to configure the plugin with `SvelteKit` defaults (you can still use the original `vite-plugin-pwa`, but you will need to configure it properly: check [SvelteKit Support](/frameworks/sveltekit#sveltekit-support)).
-
-You can check the default configuration options included by the `vite-plugin-pwa` plugin in the [SvelteKit PWA configuration module](https://github.com/antfu/vite-plugin-pwa/tree/main/src/integrations/sveltekit/config.ts).
+`Sveltkit` maintainers have been working hard to align `Vite` and `SvelteKit`. The main changes done in `SvelteKit` plugin are:
+- exposes and configures `publicDir` properly  to allow `Vite` plugins using it: `publicDir` is configured with `config.kit.files.assets` (defaults to `static` folder).
+- exposes and configures `outDir` to allow `Vite` plugins using it: `outDir` is configured with `${svelteKitOutDir}/output/client` (defaults to `.svelte-kit/output/client` folder).
+- don't call `process.exit` in the `closeBundle` hook: which means that any `Vite` plugin that is configured after the `SvelteKit` plugin (or has `enforce: 'post'`) will be called by `Vite`.
 
 To update your project to use the new `vite-plugin-pwa` for `SvelteKit`, you only need to change the `Vite` config file (you don't need oldest `pwa` and `pwa-configuration` modules):
```

Comment:
usually backticks are for code. it seems a bit unusual to me to list project names that way unless it's the repository name like `vite-plugin-pwa`
```suggestion
To update your project to use the new `vite-plugin-pwa` for SvelteKit, you only need to change the Vite config file (you don't need oldest `pwa` and `pwa-configuration` modules):
```